app/Composers/StatusPageComposer.php

```diff
@@ -62,9 +64,100 @@ public function compose(View $view)
         $componentGroups = ComponentGroup::whereIn('id', $usedComponentGroups)->orderBy('order')->get();
         $ungroupedComponents = Component::enabled()->where('group_id', 0)->orderBy('order')->orderBy('created_at')->get();
 
+        // Downtime stats for uptime table
+        $availability = array();
+
+
+        $now = Carbon::now();
+        $startOfDay=clone $now;
+        $startOfDay->setTime(0,0,0);
+        $endOfDay=clone $now;
+        $endOfDay->setTime(23,59,59);
+        $rangeToday = array();
+        array_push($rangeToday, $startOfDay);
+        array_push($rangeToday, $endOfDay);
+
+        //Today
+        $downtimeCollectionToday = Downtime::whereBetween('created_at', [$startOfDay, $endOfDay])
+                                            ->orWhereBetween('resolved_at', [$startOfDay, $endOfDay])
+                                            ->get();
+        array_push($availability, $this->calculateAvailability($downtimeCollectionToday, $rangeToday));
+
+
+        // Last week
```

Comment:
In my eyes unnecessary because the variable name is clear. But if you change the usage of the variables as I requested, maybe you can keep it.